2000-09-06  Graham Stott  <grahams@redhat.com>

	* config/i386/i386.h (ADDRESS_COST): Fix typo.

--- gcc/config/i386/i386.h.jj	Mon Oct 23 22:51:33 2000
+++ gcc/config/i386/i386.h	Fri Oct 27 17:57:35 2000
@@ -2097,7 +2097,7 @@ while (0)
    lifetimes.  */
 
 #define ADDRESS_COST(RTX) \
-  ix86_address_cost (x)
+  ix86_address_cost (RTX)
 
 /* A C expression for the cost of moving data from a register in class FROM to
    one in class TO.  The classes are expressed using the enumeration values
